packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -221,66 +284,147 @@ export class SendConfirmation extends React.Component<Props, State> {
+            />
+          )}
+          {isInvite && (
+            <LineItemRow
+              title={t('inviteFee')}
+              amount={<CurrencyDisplay amount={inviteFeeAmount} />}
+            />
+          )}
+          <LineItemRow
+            title={t('securityFee')}
+            titleIcon={<FeeIcon />}
+            amount={
+              securityFeeAmount && (
+                <CurrencyDisplay amount={securityFeeAmount} formatType={FormatType.Fee} />
+              )
+            }
+            isLoading={asyncFee.loading}
+            hasError={!!asyncFee.error}
+          />
+          <TotalLineItem amount={totalAmount} />
+        </View>
+      )
     }
 
     return (
       <SafeAreaView style={styles.container}>
         <DisconnectBanner />
         <ReviewFrame
-          HeaderComponent={this.renderHeader}
-          FooterComponent={this.renderFooter}
+          FooterComponent={renderFeeContainer}
           confirmButton={primaryBtnInfo}
-          modifyButton={secondaryBtnInfo}
           shouldReset={this.state.buttonReset}
+          isSending={this.props.isSending}
         >
-          <TransferReviewCard
-            recipient={recipient}
-            address={recipientAddress || ''}
-            e164PhoneNumber={recipient.e164PhoneNumber}
-            comment={reason}
-            value={amount}
-            currency={CURRENCY_ENUM.DOLLAR} // User can only send in Dollars
-            fee={fee}
-            isLoadingFee={asyncFee.loading}
-            feeError={asyncFee.error}
-            type={type}
-            validatedRecipientAddress={validatedRecipientAddress}
-            onEditAddressClick={this.onEditAddressClick}
-          />
-          <Modal
-            isVisible={this.state.modalVisible}
-            style={styles.modal}
-            useNativeDriver={true}
-            hideModalContentWhileAnimating={true}
-            onBackButtonPress={this.cancelModal}
-          >
+          <View style={styles.transferContainer}>
+            {isInvite && <Text style={styles.inviteText}>{t('inviteMoneyEscrow')}</Text>}
+            <View style={styles.headerContainer}>
+              <ContactCircle
+                name={this.props.transactionData.recipient.displayName}
+                thumbnailPath={getRecipientThumbnail(recipient)}
+                address={recipientAddress || ''}
+              />
+              <View style={styles.recipientInfoContainer}>
+                <Text style={styles.headerText}>Sending</Text>
+                <Text style={styles.displayName}>
+                  {getDisplayName({ recipient, recipientAddress, t })}
+                </Text>
+                {validatedRecipientAddress && (
+                  <View style={styles.editContainer}>
+                    <Text style={styles.address}>{`${validatedRecipientAddress.slice(
+                      0,
+                      6
+                    )}...${validatedRecipientAddress.slice(-4)}`}</Text>
```

Comment:
Just wondering if the designs call for this in more than one place? Should we make a ShortenedAddress component?

Reply:
Not sure but I'll create one just in case